code/controllers/subsystems/vote.dm

```diff
@@ -350,10 +350,15 @@ SUBSYSTEM_DEF(vote)
 
 
 
+
+
+/*********************
+	Storyteller
+**********************/
 /datum/poll/storyteller
 	name = "Storyteller"
 	question = "Choose storyteller"
-	time = 60
+	time = 120
```

Comment:
Should we not use MINUTES/ SECONDS define here?

Reply:
Nope because polls work on a snowflake system that takes actual seconds as input and counts them accordingly
trying to use the defines would result in times 10x longer than desired